### PR TITLE
feat: enable github integration

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -20,7 +20,11 @@ jobs:
       - name: Cypress run
         uses: cypress-io/github-action@v1
         with:
+          record: true
           build: npm run build
+        env:
+          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (ojeytonwilliams) [inline .github/workflows/cypress.yml:27]:
```suggestion
        env:
          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          record: true
          build: npm run build
```
The original order has various 'with' properties being interpreted as being part of  'env'

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

**Review by ojeytonwilliams — APPROVED** [review]
Looks good!  One thing I'm not 100% on is which branches will be watched?  If it triggers on every push and every PR I imagine we'll use up the quota.  That is unless fCC qualifies as [a non-commercial entity](https://www.cypress.io/oss-plan/).

Regardless, we might as well try this and see.

**Comment by raisedadead** [comment]
Great news!

Our OSS Application just came through:

image

**Comment by raisedadead** [comment]
I checked everything we have unlimited runs, and even then I am restricting this to pushes to `master`, `production-**` and all pull requests.

**Pull request merged**